jkube-kit/generator/api/src/test/java/org/eclipse/jkube/generator/api/support/BaseGeneratorTest.java

```diff
@@ -353,6 +353,22 @@ public void addLatestTagIfSnapshot() {
     assertTrue(tags.get(0).endsWith("latest"));
   }
 
+  @Test
+  public void addTagsFromConfig() {
+    new Expectations() {{
+        ctx.getProject();
+        result = project;
+    }};
+    BuildConfiguration.BuildConfigurationBuilder builder = BuildConfiguration.builder();
+    properties.put("jkube.generator.test-generator.tags", "tag1,tag2,latest");
```

Comment:
Would you mind covering the case when properties are like: `tag1,tag2,latest<space>1`, `tag1,tag2,latest<space>1,<space>` and `tag1,tag2,latest<space> 1<space>,bar`